e2e/rbd.go

```diff
@@ -590,7 +592,81 @@ var _ = Describe("RBD", func() {
 					e2elog.Failf("failed to validate pvc and application binding with error %v", err)
 				}
 			})
+			By("create a Block mode PVC-PVC clone and bind it to an app", func() {
```

Comment:
is there any difference between this block of code and filesystem clone? If no I would like to move the code to the helper function and make use of it to avoid much larger code duplication

Reply:
@Madhu-1 looking into the E2E , I see a good room for improvement in various things. We have to tackle it one by one. I was thinking to do that clean up or re factoring any way. I could start here or could start altogether in a new series. Please let me know the choice

Reply:
Yes, there are many things to improve in E2E. I think this would be a good start.

Reply:
Yep, I will be doing code improvement in general for the repo.. Let us start here.  Introduced a helper function.